src/ProductConstructionService/ProductConstructionService.Api/Controllers/StatusController.cs

```diff
@@ -8,29 +8,34 @@
 namespace ProductConstructionService.Api.Controllers;
 
 [Route("status")]
-public class StatusController(JobsProcessorScopeManager jobsProcessorScopeManager) : Controller
+public class StatusController(JobProcessorScopeManager jobProcessorScopeManager) : Controller
 {
-    private readonly JobsProcessorScopeManager _jobsProcessorScopeManager = jobsProcessorScopeManager;
+    private readonly JobProcessorScopeManager _jobProcessorScopeManager = jobProcessorScopeManager;
 
     [HttpPut("stop")]
-    public IActionResult StopPcsJobsProcessor()
+    public IActionResult StopPcsJobProcessor()
     {
-        _jobsProcessorScopeManager.FinishJobAndStop();
+        _jobProcessorScopeManager.FinishJobAndStop();
 
-        return GetPcsJobsProcessorStatus();
+        return GetPcsJobProcessorStatus();
     }
 
     [HttpPut("start")]
-    public IActionResult StartPcsJobsProcessor()
+    public IActionResult StartPcsJobProcessor()
     {
-        _jobsProcessorScopeManager.Start();
+        if (_jobProcessorScopeManager.State == JobsProcessorState.WaitingForVmrClone)
+        {
+            return BadRequest("The JobProcessor can't be started until the VMR is cloned");
```

Comment:
I wonder if this should be something else than `BadRequest` as that should mean your HTTP request/payload/header is wrong. It should be something like `Forbidden` or `Not Acceptable` or `Precondition Failed` etc

Reply:
I like PreconditionFailed, went with that